WalletWasabi.Backend/Global.cs

```diff
@@ -82,9 +83,9 @@ public async Task InitializeAsync(CoordinatorRoundConfig roundConfig, Cancellati
 
 		var blockNotifier = HostedServices.Get<BlockNotifier>();
 
-		bool shouldBuildCoinVerifier = CoordinatorParameters.RuntimeCoordinatorConfig.IsCoinVerifierEnabled || roundConfig.IsCoinVerifierEnabledForWW1;
```

Comment:
What happened with this guy? `IsCoinVerifierEnabledForWW1`

Reply:
`IsCoinVerifierEnabledForWW1` should be removed if we are not using it. Altough, how do we want to enable the verifier only for WW1 or -2 now?

Reply:
I'd say don't bother, just remove it. It was useful at first, but putting WW1 config in the WabiSabi config is also a smell.

Reply:
We can bring back `IsCoinVerifierEnabledForWW1` but it depends on the flags inside `WabiSabiConfig`, so it's smells a lot.
If the flags are NOT defined, but the verifier is enabled for WW1, then WW1 (same with WW2) will have a half-working verifier, which checks the addresses, then lets in the coins regardless of the answer from the API provider.

That's why I went with this. If no flags are defined, we don't use the verifier.

Alternative way would be to duplicate the flag list and paste it into the `CcjRoundConfig.json`, but honestly I do not wish to manage the same thing twice in two different file.

Reply:
We have only one source of coin verifier API, the same config, and same flags, but have two coordinators - that can have different settings. It is fine how it is now IMO.